### DISCUSSION TITLE
About the fitted light curve

### DISCUSSION THREAD

**Discussion by hurran123** [sncosmo/sncosmo#392] [Q&A]
I have fitted some data points using the fit_lc function, how do I extract the fitted function? I am using the salt2 model.

**Comment by benjaminrose** [comment]
Are you wanting the parameter values? (results.parameters should work, https://sncosmo.readthedocs.io/en/stable/examples/plot_lc_fit.html#sphx-glr-examples-plot-lc-fit-py)

Or are you wanting the function so you can, say, extract the flux at an arbitrary phase? (Send results.parameters to a new salt2 model, https://sncosmo.readthedocs.io/en/stable/models.html)
